src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/client/ApiClientServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.client;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.ApiClient;
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.api.sdk.ApiClientService;
+import uk.gov.companieshouse.sdk.manager.ApiSdkManager;
+
+import java.io.IOException;
+
+@Component
+public class ApiClientServiceImpl implements ApiClientService {
```

Comment:
Just wondering where this class is being used and how? I can't see any references to it.

Reply:
This implements an interface so requires all 4 methods to be implemented or the compiler will complain.

You won't see references it is an implementation of a interface referred to in the library. I recommend looking at the TransactionInterceptor class in the library. This library requires an implementation otherwise the app will not start.

Reply:
👍 makes sense now. Thanks.